frontend/src/Student/pages/Dashboard.js

```diff
@@ -1,17 +1,22 @@
 import React from 'react'
 import { Container, Row, Col } from 'reactstrap'
+import { connect } from 'react-redux'
 
 import Shows from '../containers/Shows'
 
-const Dashboard = () => (
+const Dashboard = props => (
   <Container>
     <Row>
       <Col>
-        <Shows />
+        <Shows studentUsername={props.studentUsername}/>
```

Comment:
I'm not sure why you need this change here. In `containers/Shows.js` the `studentUsername` prop is already obtained in `mapStateToProps`. Passing the prop in again (and redefining the same `mapStateToProps`here ) seems redundant.

Reply:
When I tried removing it, the property was empty when it attempted the query

Reply:
Let me test this out before you merge it in – that seems odd.